src/components/Auth/SignUp.tsx

```diff
@@ -84,75 +86,116 @@ const SignUp = ({ invite }: SignupProps) => {
-              },
-            }}
-            isDisabled={!!invite}
-          />
-          <InputPassword
-            formValue='password'
-            label={t('password')}
-            placeholder={t('password_placeholder')}
-            type='password'
-            validation={{
-              required: t('form.error.field_is_required'),
-              minLength: {
-                value: 8,
-                message: t('form.error.password_min_length', { defaultValue: 'Min. 8 characters' }),
-              },
-            }}
-          />
-          <FormControl as='fieldset' isInvalid={!!errors?.terms}>
-            <Checkbox {...register('terms', { required: t('cc.validation.required') })} isRequired>
-              <Trans
-                i18nKey='signup_agree_terms'
-                components={{
-                  termsLink: <Link isExternal as={ReactRouterLink} to={Routes.terms} />,
-                  privacyLink: <Link isExternal as={ReactRouterLink} to={Routes.privacy} />,
-                }}
+          <Flex flexDirection={'column'} gap={4} mb={4}>
+            <InputBasic formValue='firstName' label={t('signup_name')} placeholder={'John Doe'} />
+            <InputBasic
+              formValue='email'
+              label={t('email')}
+              placeholder={t('email_placeholder', { defaultValue: 'your@email.com' })}
+              validation={{
+                required: t('form.error.field_is_required'),
+                pattern: {
+                  value: /^[^\s@]+@[^\s@]+\.[^\s@]+$/, // Regex para validar emails
+                  message: t('form.error.email_invalid', { defaultValue: 'Invalid email address' }),
+                },
+              }}
+              isDisabled={!!invite}
+            />
+            <InputPassword
+              formValue='password'
+              label={t('password')}
+              placeholder={'• • • • • • • •'}
+              type='password'
+              validation={{
+                required: t('form.error.field_is_required'),
+                minLength: {
+                  value: 8,
+                  message: t('form.error.password_min_length', { defaultValue: 'Min. 8 characters' }),
+                },
+              }}
+            />
+            <InputPassword
+              formValue='confirm_password'
+              label={t('confirm_password')}
+              placeholder={'• • • • • • • •'}
+              type='password'
+              validation={{
+                required: t('form.error.field_is_required'),
+                validate: (value) => value === password || t('passwords_do_not_match'),
+              }}
+            />
+            <FormControl as='fieldset' isInvalid={!!errors?.terms}>
+              <Checkbox
+                {...register('terms', { required: t('cc.validation.required') })}
+                color='auth.checkboxes_text'
+                isRequired
```

Comment:
Remove the `isRequired`, since right now this is the only error you'll see initially until marked. Removing the `isRequired` (but keeping the validation) gives a better UX.